packages/sanity/src/core/form/inputs/StringInput/StringInputPortableText/StringInputPortableText.tsx

```diff
@@ -132,7 +132,7 @@ export function StringInputPortableText(props: StringInputProps) {
     [onFocus, onBlur, onOptimisticChange, definitiveValue, onChange],
   )
 
-  const initialConfig = useRef<EditorConfig>({
+  const [initialConfig] = useState<EditorConfig>(() => ({
```

Comment:
`useState` is a better choice for capturing a stable value for the lifetime of a component. Unlike `useRef` it's always safe to read `useState()[0]` during render.